acvm_backend_barretenberg/src/acvm_interop/pwg.rs

```diff
@@ -162,3 +165,279 @@ impl PartialWitnessGenerator for Barretenberg {
         Ok(OpcodeResolution::Solved)
     }
 }
+
+// TODO: alter this method so that it only processes one hash per level rather than overriding
+// the one of leaves for each level of the hash path
```

Comment:
```suggestion
```

This comment looks stale to me as it looks like we're doing the minimal number of hashes.

Reply:
Yes it is stale